src/renderer/features/governance/components/ReferendumDetails/ReferendumAdditional.tsx

```diff
@@ -0,0 +1,39 @@
+import { ReferendumId } from '@shared/core';
+import { FootnoteText, Icon } from '@shared/ui';
+import { useI18n } from '@app/providers';
+
+type Props = {
+  referendumId: ReferendumId;
+  network: string;
+};
+
+export const ReferendumAdditional = ({ network, referendumId }: Props) => {
+  const { t } = useI18n();
+
+  return (
+    <div className="flex flex-col gap-3">
+      <a
+        className="flex items-center gap-1.5 px-1.5 py-1"
+        href={t('governance.additional.polkassemblyLink', { network, referendumId })}
```

Comment:
What is the reason to use i18n for links? Never use it such way before

Reply:
i18n eslint plugin throws error on static string, unfortunately 🫠